CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,62 @@
+<div id="theia-logo" align="left">
+    <br />
+    <img src="https://raw.githubusercontent.com/eclipse-theia/theia/master/logo/EF_GRY-OR_svg.svg?sanitize=true" alt="Eclipse Logo" width="300"/>
+</div>
+
+# Community Code of Conduct
+
+Version 1.1
+
+October 21, 2019
```

Comment:
Jonas, this seems like a really old version. I suggest we use the Jan 1 2023 version:
https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php

If the direct link doesn't work, you can get there through the EF project dashboard -> legal documentation generator.